benchmark.h

```diff
@@ -348,6 +421,7 @@ class Benchmark {
   bool measure_each;
   bool cold_cache;
   bool track_errors;
+  bool csv;
```

Comment:
Nit: private members should end with `_`. Feel free to rename `measure_each` etc. as well.